app/controllers/allocations_controller.rb

```diff
@@ -20,7 +20,7 @@ def new
   # rubocop:disable Metrics/LineLength
   def show
     @prisoner = offender(nomis_offender_id_from_url)
-    primary_pom_nomis_id = AllocationService.primary_pom_nomis_id(@prisoner.offender_no)
+    primary_pom_nomis_id = Allocation.primary_pom_nomis_id(@prisoner.offender_no)
```

Comment:
I wonder if this might be better as 
```
primary, secondary = AllocationService.get_poms(allocation_id) 
```

Reply:
Yep that makes sense :-)  - perhaps hold fire on the change until Secondary POMs are introduced (next PR)?